docs/Tutorials/OpenAPI.md

```diff
@@ -1,9 +1,17 @@
 # OpenAPI
 
-Pode has inbuilt support for converting your routes into OpenAPI 3.0 definitions. There is also support for a enabling simple Swagger and/or ReDoc viewers.
+
+Pode has built-in support for converting your routes into OpenAPI 3.0 definitions. There is also support for enabling simple Swagger and/or ReDoc viewers and others.
+
+The OpenApi module has been extended with many more functions, and some old ones have been improved. Anyway, the module is backward compatible. Please refer to the old documentation [OpenApi Tutorial 2.9](../OpenApi/OpenAPI2_9.md) for the old syntax.
```

Comment:
Sine the new logic is all backwards compatible, there's no need for this line :)

Reply:
yes it's backwards compatible but all samples are with the new syntax using pipeline. I don't have any sample using the old syntax